docs/howto/delete-app.txt

```diff
@@ -0,0 +1,31 @@
+==========================
+How to delete a Django app
+==========================
+
+Before deleting an application from your project you should consider that the
+app A that you are about to delete may have migrations on which app B depends
+and in case you want to remove B in the future you may also encounter an error
+like this when you want to revert migrations or running a :djadmin:`squashmigrations`:
+
+.. code-block:: shell
+
+    django.db.migrations.exceptions.NodeNotFoundError: Migration
+    <app_B>.XXXX_migration_name dependencies reference
+    nonexistent parent node ('<app_A>', 'XXXX_migration_name')
+
+Once this is said, the steps to follow are:
```

Comment:
```suggestion
```
I find this preamble to be confusing to follow. I reckon it's definitely enough to just add a step to say "squash out references to the app in other apps' migrations".

Reply:
Agree and deleted!